app/default/routes.py

```diff
@@ -23,15 +25,44 @@
 @default_bp.route("/")
 def index():
     current_app.logger.info("Service landing page loaded.")
+    round_data = get_round_data(Config.DEFAULT_FUND_ID, Config.DEFAULT_ROUND_ID, as_dict=True)
+    submission_deadline = datetime.strptime(round_data.deadline, "%Y-%m-%d %X").strftime("%d %B %Y")
     return render_template(
-        "index.html", service_url=Config.ENTER_APPLICATION_URL
+        "index.html",
+        service_url=Config.ENTER_APPLICATION_URL,
+        submission_deadline=submission_deadline,
+        contact_us_email_address=round_data.contact_details["email_address"]
     )
 
 
 @default_bp.route("/accessibility_statement", methods=["GET"])
 def accessibility_statement():
     current_app.logger.info("Accessibility statement page loaded.")
-    return render_template("accessibility-statement.html")
+    return render_template("accessibility_statement.html")
+
+
+@default_bp.route("/cof-r2w2-all-questions", methods=["GET"])
+def all_questions():
+    current_app.logger.info("All questions page loaded.")
+    round_data = get_round_data(Config.DEFAULT_FUND_ID, Config.DEFAULT_ROUND_ID, as_dict=True)
```

Comment:
A bit nitpicky... but do we need the network call here? If the whole page is hardcoded questions, and called `cof-r2w2-etc` can we just hardcode the round name too? Then this is just static content and we don't need to `render_template`

Reply:
Yep, good spot! Fixed.